### PR TITLE
LPD-28058 - Filter user-accounts by status

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-28058

### CODE REVIEW

**Comment by matyaswollner** [comment]
ci:test:sf

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/odata/entity/v1_0/UserAccountEntityModel.java [hunk 1]
```diff
@@ -63,7 +64,8 @@ public UserAccountEntityModel() {
 				"givenName", locale -> Field.getSortableFieldName("firstName")),
 			new StringEntityField(
 				"jobTitle", locale -> Field.getSortableFieldName("jobTitle")),
-			new StringEntityField("name", locale -> Field.USER_NAME));
+			new StringEntityField("name", locale -> Field.USER_NAME),
+			new IntegerEntityField("status", locale -> Field.STATUS));
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/odata/entity/v1_0/UserAccountEntityModel.java:68]: please move it in the middle to be sorted
```
new IdEntityField
new IntegerEntityField
new StringEntityField
```

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by smottal** [comment]
@matyaswollner only a sort. Thx

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**